### ISSUE TITLE
multiple processes crawling the same endpoint (blocked by #183)

### ISSUE THREAD

**Issue by SaravShah** [sul-dlss/preservation_catalog#174]
-This audit task should be shardable on a per-endpoint basis (multiple processes crawling the same endpoint, investigating different portions of the druid tree without contention).
- be able to partition with druid name space so maybe like have two instances of audit running one doing the first half or druidss like aa-jj then another instance doing the 2nd half like jj-zz druids??

**Comment by ndushay** [comment]
pre-req for #68 (or at least related to it)

**Comment by ndushay** [comment]
do we need to worry about shardable yet?  I'm inclined to say no.

Do we want a ticket for "multiple roots", and another for "shardable for single endpoint"?   I'd argue for the first being critical path this go round, and the latter being in the backlog ...

**Comment by jmartin-sul** [comment]
this is no longer relevant now that audit checks have been moved to a queue and worker setup
